imageregistry/v1/00-crd.yaml

```diff
@@ -11,6 +11,7 @@ spec:
     listKind: ConfigList
     plural: configs
     singular: config
+  preserveUnknownFields: false
```

Comment:
@dmage @adambkaplan We were not setting this option to `false` so `oc explain` command was not working for this CRD. Is this OK or do we  have some "undocumented"(not defined on the CRD) fields we do use?